_includes/catalogpage.html

```diff
@@ -22,9 +22,7 @@
         {% if comp.categories contains include.category %}
         <li class="cards__item">
             <div class="catalog-entry">
-                <div class="catalog-image-holder">
-                    {% if comp.image contains '<svg' %}{{comp.image}}{% else %}<img alt="" src="{{comp.image}}" />{% endif %}
-                </div>
+                <div class="catalog-image-holder">{{comp.image}}</div>
```

Comment:
@Hixie Images can be SVG or png's etc, so this would render just the url in many cases